beacon-chain/powchain/service.go

```diff
@@ -26,6 +28,25 @@ import (
 
 var log = logrus.WithField("prefix", "powchain")
 
+var (
+	validDepositsCount = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "powchain_valid_deposits_received",
+		Help: "The number of valid deposits received in the deposit contract",
+	})
+	totalDepositsCount = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "powchain_deposit_logs",
+		Help: "The total number of deposits received in the deposit contract",
+	})
+	chainStartCount = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "powchain_chainstart_logs",
+		Help: "The number of chainstart logs received from the deposit contract",
```

Comment:
only one chain start will happen(it is boolean) , shouldnt this be more in the line of ‘Has chain start log filed?’

Reply:
It should only happen once, but if it happens more than once, then that is a very interesting metric to alert. Don't you think? :)

Reply:
Think: reorg, or something

Reply:
True true. Let's hope that never happens 🙏🏼